public/app/plugins/panel/nodeGraph/NodeGraph.tsx

```diff
@@ -208,6 +226,26 @@ export function NodeGraph({ getLinks, dataFrames, nodeLimit, panelId, zoomMode }
         </div>
       ) : null}
 
+      {!layoutAlgorithm && (
+        <div className={styles.layoutAlgorithm}>
```

Comment:
Nit: In graph/timeseries panel options are added to the container's PanelChrome actions ([code](https://github.com/grafana/grafana/blob/v11.6.0/public/app/features/explore/Graph/GraphContainer.tsx#L94)). This makes it aligned to the top right corner. Looks a bit cleaner and saves more space:

| timeseries | node graph |
|---|---|
| <img width="995" alt="Screenshot 2025-03-26 at 13 29 57" src="https://github.com/user-attachments/assets/9d2ae2bd-7b68-40a8-bb60-19defc1fdec3" /> | <img width="996" alt="Screenshot 2025-03-26 at 13 30 09" src="https://github.com/user-attachments/assets/b0d5f49a-649d-45c1-a8f4-226cd1807077" /> |